### DISCUSSION TITLE
heif: Unsupported feature: Unsupported codec (4.3000)

### DISCUSSION THREAD

**Discussion by Charltsing** [libvips/libvips#3407] [General]
NetVips.VipsException
  HResult=0x80131500
  Message=unable to call VipsForeignSaveSpngTarget
C:\xxx\pic.HEIC: bad seek to 998149
heif: Unsupported feature: Unsupported codec (4.3000)

https://charltsing.lanzoum.com/iUQkB0r16reb

ImageMagick work fine.

**Comment by lovell** [comment]
>  heif: Unsupported feature: Unsupported codec (4.3000)

It looks like you're using a version of libvips+libheif compiled without support for libde265. The HEVC codec is patent encumbered so you'll probably need to build everything from source if you require this feature. https://github.com/libvips/libvips#building-from-source
